src/components/cylc/gscan/GScan.vue

```diff
@@ -263,29 +263,19 @@ export default {
       filters: [
         {
           title: 'workflow state',
-          items: [
-            {
-              text: 'running',
-              value: 'running',
-              model: true
-            },
-            {
-              text: 'held',
-              value: 'held',
-              model: true
-            },
-            {
-              text: 'stopped',
-              value: 'stopped',
+          items: WorkflowState.enumValues.map(state => {
+            return {
+              text: state.name,
+              value: state,
               model: true
             }
-          ]
+          })
```

Comment:
We were not displaying a few values (added afterwards in the server side I think?). Using the enum it should be easier to keep everything up to date. We just need to remember to update the enums.

Reply:
And also returning the enum as value. We were doing that in the `TaskState` filter, no reason for just repeating the string here. This way we are more consistent, and it's also easier to write tests, and less likely to make mistakes in the code (as I did a few minutes ago, and then had to spend ~15 minutes chasing what was wrong; I had used `.state.name` by accident in the wrong filter)